Vision/src/V1/ImageAnnotatorClient.php

```diff
@@ -515,11 +515,24 @@ public function objectLocalization($image, $optionalArgs = [])
      *
      * Example:
      * ```
+     * use Google\Cloud\Vision\V1\ProductSearchClient;
+     * use Google\Cloud\Vision\V1\ProductSearchParams;
+     *
      * $imageContent = file_get_contents('path/to/image.jpg');
-     * $response = $imageAnnotatorClient->productSearch($imageContent);
+     * $productSetName = ProductSearchClient::productSetName('PROJECT_ID', 'LOC_ID', 'PRODUCT_SET_ID');
+     * $productSearchParams = (new ProductSearchParams)
+     *     ->setProductSet($productSetName);
+     * $response = $imageAnnotatorClient->productSearch(
+     *     $imageContent,
+     *     $productSearchParams
+     * );
      * ```
      *
      * @param resource|string|Image $image The image to be processed.
+     * @param ProductSearchParams   $productSearchParams Parameters for a product search request. Please note, if the
```

Comment:
I had to read twice to parse the sentence right, because I needed to figure out what `this value` points to (ruling out the optional image context argument).

How about just:

```
Please note, this value will override the ProductSearchParams in the {@see Google\Cloud\Vision\V1\ImageContext} instance if provided.
```

or something?

Reply:
👍 that sounds cleaner